delocate/tools.py

```diff
@@ -393,14 +393,17 @@ def find_package_dirs(root_path):
     Returns
     -------
     package_sdirs : set
-        Set of strings where each is a subdirectory of `root_path`, containing
-        an ``__init__.py`` file.  Paths prefixed by `root_path`
+        Set of strings where each is a subdirectory of `root_path` containing an
+        ``__init__.py`` file, or a C extension single-file module directly in
+        `root_path`.  Paths prefixed by `root_path`
     """
     package_sdirs = set()
     for entry in os.listdir(root_path):
         fname = entry if root_path == '.' else pjoin(root_path, entry)
         if isdir(fname) and exists(pjoin(fname, '__init__.py')):
-            package_sdirs.add(fname)
+            package_sdirs.add((fname, True))
+        elif isfile(fname) and ((fname.endswith('.so') or fname.endswith('.dylib'))):
```

Comment:
Use `lib_filt_func` here?  See `delocate_wheel` docstring.

Reply:
I'm sure sure I understand, I might have been too hasty but `lib_filt_func` is to decide what files to consider when looking for libraries.  I guess it's fine print.   Yes, moving to `tools` sounds like a reasonable thing to do.